gulpfile.js

```diff
@@ -68,61 +73,64 @@ var config = {
   }
 };
 
-gulp.task('build:js', ['build:js:unmin', 'build:js:min']);
-
-gulp.task('build:js:unmin', function () {
+function jsNotMin() {
   return browserify(config.src.js.main, {standalone: 'braintree.dropin'})
     .bundle()
     .pipe(source(config.src.js.output))
     .pipe(replace('@DOT_MIN', ''))
     .pipe(streamify(size()))
-    .pipe(gulp.dest(config.dist.js))
-});
+    .pipe(dest(config.dist.js));
+}
 
-gulp.task('build:js:min', function () {
+function jsMin() {
   return browserify(config.src.js.main, {standalone: 'braintree.dropin'})
     .bundle()
     .pipe(source(config.src.js.output))
     .pipe(replace('@DOT_MIN', '.min'))
     .pipe(streamify(uglify()))
     .pipe(streamify(size()))
     .pipe(rename(config.src.js.min))
-    .pipe(gulp.dest(config.dist.js));
-});
+    .pipe(dest(config.dist.js));
+}
 
-gulp.task('build:css', function () {
+jsNotMin.displayName = 'build:js:notmin';
+jsMin.displayName = 'build:js:min';
+
+function buildCss() {
   var lessOptions = {};
 
-  return gulp.src(config.src.css.main)
-    .pipe(less(lessOptions)).on('error', function(error) {
```

Comment:
i had added this error handling in #502 mistakenly thinking it was required doing a 1:1 replacement for Sass, but apparently gulp-less is already handling its errors properly (as i discovered when digging around in source code)